packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentInformation.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import { mergeClasses } from '../../../classify';
 
+import PriceAdjustments from '../PriceAdjustments';
+import Button from '../../Button';
 import defaultClasses from './paymentInformation.css';
 
 const PaymentInformation = props => {
-    const { showContent, doneEditing } = props;
+    const { onSave } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    // TODO: Replace "doneEditing" with a query for existing data.
+    const [doneEditing, setDoneEditing] = useState(false);
```

Comment:
> Shouldn't these couple lines of code go into a talon?

Yep! These were just temporary additions. I assumed whomever took the component ticket would move the logic for this component into a talon.

> And I am not sure if you can replace doneEditing for a data query because the presence of data is not the same as the completion of the step which doneEditing signifies.

This state was simply meant to signal whether to render in "edit mode" or in "read only mode". If the cart has payment information already, this component should render as if it was in read only mode. Same for the rest of the components.

Reply:
If the data is cart scoped then we are good.